src/%ZPM/PackageManager/Client/Settings.cls

```diff
@@ -22,7 +22,19 @@ Parameter TerminalPrompt = "TerminalPrompt";
 
 Parameter PublishTimeout = "publish_timeout";
 
-Parameter CONFIGURABLE = "trackingId,analytics,ColorScheme,TerminalPrompt,PublishTimeout";
+/// A path pointing either to a python executable or a pip executable, controlled by #UseStandalonePip
```

Comment:
Can use `<parameter>UseStandalonePip</parameter>` so it renders with a link in the class reference.